doc/connectivity/bluetooth/bluetooth-shell.rst

```diff
@@ -0,0 +1,357 @@
+        [DEVICE]: E3:72:76:87:2F:E8 (random), AD evt type 3, RSSI -74  C:0 S:0 D:0 SR:0 E:0 Prim: LE 1M, Secn: No packets, Interval: 0x0000 (0 us), SID: 0xff
+        [DEVICE]: 1E:19:25:8A:CB:84 (random), AD evt type 3, RSSI -67  C:0 S:0 D:0 SR:0 E:0 Prim: LE 1M, Secn: No packets, Interval: 0x0000 (0 us), SID: 0xff
+        [DEVICE]: 26:42:F3:D5:A0:86 (random), AD evt type 3, RSSI -73  C:0 S:0 D:0 SR:0 E:0 Prim: LE 1M, Secn: No packets, Interval: 0x0000 (0 us), SID: 0xff
+        [DEVICE]: 0C:61:D1:B9:5D:9E (random), AD evt type 3, RSSI -87  C:0 S:0 D:0 SR:0 E:0 Prim: LE 1M, Secn: No packets, Interval: 0x0000 (0 us), SID: 0xff
+        [DEVICE]: 20:C2:EE:59:85:5B (random), AD evt type 3, RSSI -66  C:0 S:0 D:0 SR:0 E:0 Prim: LE 1M, Secn: No packets, Interval: 0x0000 (0 us), SID: 0xff
+        [DEVICE]: 25:3F:7A:EE:0F:55 (random), AD evt type 3, RSSI -83  C:0 S:0 D:0 SR:0 E:0 Prim: LE 1M, Secn: No packets, Interval: 0x0000 (0 us), SID: 0xff
+        uart:~$ bt scan off
+        Scan successfully stopped
+
+As you can see, this can lead to a high number of results. To reduce that number and easily find a
+specific device, you can enable scan filters. There are four types of filtering, by name, by RSSI,
+by address and by periodic advertising interval. To apply a filter, you need to use
+:code:`bt scan-set-filter` followed by the type of filtering.
+
+For example, if you want to look only for device with the name *test shell*:
+
+.. code-block:: console
+
+        uart:~$ bt scan-filter-set name "test shell"
+
+Or if you want to look for devices on really close range:
+
+.. code-block:: console
+
+        uart:~$ bt scan-filter-set rssi -40
+        RSSI cutoff set at -40 dB
+
+Finally, if you want to remove all filters:
+
+.. code-block:: console
+
+        uart:~$ bt scan-filter-clear all
+
+Connecting to a device
+**********************
+
+To connect to a device, if you know its address and type of address simply use :code:`bt connect`
+with the address and the type as arguments.
+
+Here is an example:
+
+.. code-block:: console
+
+        uart:~$ bt connect 52:84:F6:BD:CE:48 random
+        Connection pending
+        Connected: 52:84:F6:BD:CE:48 (random)
+        Remote LMP version 5.3 (0x0c) subversion 0xffff manufacturer 0x05f1
+        LE Features: 0x000000000001412f
+        LE PHY updated: TX PHY LE 2M, RX PHY LE 2M
+        LE conn  param req: int (0x0018, 0x0028) lat 0 to 42
+        LE conn param updated: int 0x0028 lat 0 to 42
+
+You can list the active connections of the shell using :code:`bt connections`. The shell maximum
+number of connections is defined by :kconfig:option:`CONFIG_BT_MAX_CONN`. You can disconnect from a
+connection with :code:`bt disconnect <address: XX:XX:XX:XX:XX:XX> <type: (public|random)>`.
+
+On a side note, if you were scanning just before, you can connect to the last scanned device by
```

Comment:
There is a sphinx tag for notes (`.. note::`), see the getting started .rst for usage.
Suggest removing `on a side note,` and using that tag instead.